File: C/unit_2/Fg.c
#include <stdio.h>
#include <math.h>
int main()
{
  float ang, s, c, t, sc, csc, ct;
  printf("Enter the Angle : \n");
  scanf("%f", &ang);
  s = sin(ang);
  c = cos(ang);
  t = tan(ang);
  sc = acos(ang);
  csc = asin(ang);
  ct = atan(ang);
  printf("Sine Value is : %f \n Cosine Value is : %f \n Tangent Value is : %f \n Secant Value is : %f \n Cosecant Value is : %f \n Cotangent Value is : %d", s, c, t, sc, csc, ct);
  return 0;
}

File: Java/unit_2/Fg.java
package Java.unit_2;

import java.util.Scanner;

public class Fg {
    public static void main(String[] args) {
        double ang, s, c, t, sci, csc, ct;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Angle : ");
        ang = sc.nextDouble();
        s = Math.sin(ang);
        c = Math.cos(ang);
        t = Math.tan(ang);
        sci = Math.acos(ang);
        csc = Math.asin(ang);
        ct = Math.atan(ang);
        System.out.println("Sine Value is : " + s);
        System.out.println("Cosine Value is : " + c);
        System.out.println("Tangent Value is : " + t);
        System.out.println("Secant Value is : " + sci);
        System.out.println("Cosecant Value is : " + csc);
        System.out.println("Cotangent Value is : " + ct);
        sc.close();
    }
}
